util-images/network/netperfbenchmark/pkg/worker/util.go

```diff
@@ -66,31 +68,29 @@ var (
 func parseResult(protocol string, result []string, testDuration string) ([]float64, error) {
 	switch protocol {
 	case ProtocolTCP:
-		return parseIperfResponse(result, testDuration, tcpMetricsCount, ProtocolTCP, zeroFormatTCP, fractionZeroFormatTCP, iperfTCPFunction)
+		return parseIperfResponse(result, testDuration, tcpMetricsCount, ProtocolTCP, iperfTCPFunction)
 	case ProtocolUDP:
-		return parseIperfResponse(result, testDuration, udpMetricsCount, ProtocolUDP, zeroFormatUDP, fractionZeroFormatUDP, iperfUDPFunction)
+		return parseIperfResponse(result, testDuration, udpMetricsCount, ProtocolUDP, iperfUDPFunction)
 	case ProtocolHTTP:
 		return parseSiegeResponse(result)
 	default:
 		return nil, errors.New("invalid protocol: " + protocol)
 	}
 }
 
-func parseIperfResponse(result []string, testDuration string, metricCount int, protocol, zeroFormat, fractionZeroFormat string, operators []string) ([]float64, error) {
+func parseIperfResponse(result []string, testDuration string, metricCount int, protocol string, operators []string) ([]float64, error) {
 	aggregatedResult := make([]float64, 0, metricCount)
 	count := 0
 	sessionID := make(map[string]bool)
-	interval := zeroFormat + "-" + testDuration + fractionZeroFormat
 	for _, line := range result {
+		klog.V(4).Info(line)
```

Comment:
Is it ok if we keep it for first few runs and then later remove?, the command parsing etc should work as we haven't seen any issue's until now, but incase we need to debug ,this would be the only way.